### DISCUSSION TITLE
Don't understand how to make work the « Quick start » Scene in the THNK example

### DISCUSSION THREAD

**Discussion by Angelars77** [arthuro555/THNK#32] [General]
Hello Arthuro ! Please i'm a newbie and i don't know how to make work the ServerQuickStart and the ClientQuickStart Scenes in your THNK example in GDevelop.

I tried to preview the first before the second, while connected to the internet, but it displayed an error like : A JavaScript error occurred in the main process / Uncaught Exception : Error : EPERM : operation not permitted, open 'C:\Program Files\GDevelop\geckos-server.zip'

I have put an eye in this and i noticed that i don't have the geckos-server.zip file.

What should i do please ?

**Comment by arthuro555** [comment]
You are on an old version of geckos. geckos-server.zip is an automatically downloaded file, but since you installed GDevelop in a protected folder, it cannot be downloaded there without administrator permissions. Newer versions of the geckos adapter download the server to a folder that does not require administrator permissions instead of the GDevelop directory.

**Reply by Angelars77** [reply]
THaNKs Arthuro
But i've another problem 😅
I've made work the « ServerQuickStart » scene by opening GDevelop as admin but the « ClientQuickStart » scene stays grey, like the client is not connected to the server. 
What should i do to fix it please ?

**Reply by arthuro555** [reply]
- Check the console for errors
- Make sure all THNK extensions are up-to-date
- Make sure your connection events are correct
- Make sure that you use a 4 digits port number
- Make sure you are using the adapter with a thnk scene

That's all the troubleshooting tips i can think of on the top of my mind 🤔 If you cannot get the geckos adapter to work, try to use the P2P one instead.
